lib/internal/policy/manifest.js

```diff
@@ -547,3 +618,107 @@ ObjectSetPrototypeOf(Manifest.prototype, null);
 ObjectFreeze(Manifest);
 ObjectFreeze(Manifest.prototype);
 module.exports = ObjectFreeze({ Manifest });
+
+// #region URL utils
+
+/**
+ * Attempts to canonicalize relative URL strings against a base URL string
+ * Does not perform I/O
+ * If not able to canonicalize, returns the original specifier
+ *
+ * This effectively removes the possibility of the return value being a relative
+ * URL string
+ * @param {string} specifier
+ * @param {string} base
+ * @returns {string}
+ */
+function canonicalizeSpecifier(specifier, base) {
+  try {
+    if (RegExpPrototypeTest(kRelativeURLStringPattern, specifier)) {
+      return resolve(specifier, base).href;
+    }
+    return resolve(specifier).href;
+  } catch {}
```

Comment:
Why are you catching here but not doing anything with the error?

Reply:
matching : https://wicg.github.io/import-maps/#sort-and-normalize-a-specifier-map

this basically makes it soo that relative strings get resolved, but "bare" specifiers are able to be intercepted (rather than error)